src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -1214,15 +1214,15 @@ bool QgsPostgresProvider::loadFields()
     {
       const QString seqName { mTableName + '_' + fieldName + QStringLiteral( "_seq" ) };
       const QString seqSql { QStringLiteral( "SELECT c.oid "
```

Comment:
Let's drop the initialiser brace use here -- it's a constant source of issues with astyle

Reply:
done, but that doesn't change astyle output...